net/shadowsocks-libev/files/ss-rules

```diff
@@ -92,18 +92,26 @@ ac_rule() {
 		fi
 	fi
 
+	ipset -! -R <<-EOF || return 1
+		create ss_spec_lan_ac hash:net
+		$(for ip in ${LAN_AC_IP:1}; do echo "add ss_spec_lan_ac $ip $TAG"; done)
+EOF
+
 	ROUTECHAIN=PREROUTING
 	if iptables-save -t nat | grep -q "^:zone_lan_prerouting"; then
 		ROUTECHAIN=zone_lan_prerouting
 	fi
 
-	ipset -! -R <<-EOF || return 1
-		create ss_spec_lan_ac hash:net
-		$(for ip in ${LAN_AC_IP:1}; do echo "add ss_spec_lan_ac $ip $TAG"; done)
-EOF
 	$ipt_n -A $ROUTECHAIN -p tcp $EXT_ARGS \
-		-m set ! --match-set ss_spec_lan_ac src \
-		-m comment --comment "_SS_SPEC_RULE_" -j SS_SPEC_WAN_AC
+	       -m set ! --match-set ss_spec_lan_ac src \
+	       -m comment --comment "_SS_SPEC_RULE_" -j SS_SPEC_WAN_AC
+
+	if iptables-save -t nat | grep -q "^:zone_wan_prerouting"; then
+		ROUTECHAIN=zone_wan_prerouting
+		$ipt_n -A $ROUTECHAIN -p tcp $EXT_ARGS \
+		       -m set ! --match-set ss_spec_lan_ac src \
+		       -m comment --comment "_SS_SPEC_RULE_" -j SS_SPEC_WAN_AC
+	fi
```

Comment:
This seems to be just hack code which I personally think is not ready yet for inclusion into this repo.

Other than that, the package has now been rewritten in the `master` branch.  Please consider giving that a try (rules will be inserted to the front of `PREROUTING` chain).